encoding/codecv7_test.go

```diff
@@ -384,18 +384,6 @@ func TestCodecV7BatchStandardTestCasesEnableCompression(t *testing.T) {
 		})
 	}
 
-	repeat := func(element byte, count int) string {
```

Comment:
Why we should erase this testing case?

Reply:
because it's an unable to compress test case. or we can add it back and change the versioned hash. wdyt?